### PR TITLE
fix directory

### PR DESCRIPTION
**Pull request by habibayassin**

### CODE REVIEW

File: etc/test_delta_debug.sh [hunk 1]
```diff
@@ -5,7 +5,8 @@
 # variable names such as $(false), unused variables, etc.
 set -ue -o pipefail
 
-cd "$(dirname $(readlink -f $0))/../flow"
+cd "$(dirname $(readlink -f $0))"
+cd ../../../flow
```
Comment (vvbandeira) [inline etc/test_delta_debug.sh:9]: Is this script exclusive to ORFS?

**Comment by vvbandeira** [comment]
@oharboe
I might have missed the discussion, but why is this script in the OR and not ORFS repo?

**Comment by maliberty** [comment]
It can be used to reduce any OR test case, not just ORFS.

File: etc/test_delta_debug.sh (hunk 1 above)
Comment (habibayassin) [reply etc/test_delta_debug.sh:9]: it seems like it is mainly targeting ORFS

Comment (oharboe) [reply etc/test_delta_debug.sh:9]: To clarify: this integration test is testing the generic deltaDebug.py, which is independent of ORFS, using test-data from ORFS.

Comment (oharboe) [reply etc/test_delta_debug.sh:9]: Perhaps the integration test of deltaDebug.py could be moved to ORFS? Lots of OpenROAD integration testing happens in ORFS.

**Review by vvbandeira — APPROVED** [review]

**Pull request merged**